<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDial" name="RPM">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="maximum">
     <number>7000</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RPM</string>
    </property>
   </widget>
   <widget class="QLabel" name="RPM_r">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QDial" name="speed">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>160</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle1_r">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>510</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QDial" name="angle1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Angle1</string>
    </property>
   </widget>
   <widget class="QDial" name="angle2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>380</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>510</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Angle2</string>
    </property>
   </widget>
   <widget class="QLabel" name="angle2_r">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>520</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QDial" name="temp">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>340</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>temp</string>
    </property>
   </widget>
   <widget class="QLabel" name="temp_r">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QDial" name="fuel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="fuel_r">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>330</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>330</y>
      <width>64</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>fuel</string>
    </property>
   </widget>
   <widget class="QLabel" name="speed_r">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>160</y>
      <width>64</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="left">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>560</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>left</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="highs">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>590</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>highs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="seatbelt">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>590</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>seatbelt</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="engine">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>620</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>engine</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="low">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>650</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>low</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fourWheel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>620</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>fourWheel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="right">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>560</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>right</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="brake">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>680</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>brake</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="oil">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>680</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>oil</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="batt">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>650</y>
      <width>93</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>batt</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>RPM</sender>
   <signal>valueChanged(int)</signal>
   <receiver>RPM_r</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>speed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>speed_r</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>381</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angle1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>angle1_r</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>angle2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>angle2_r</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>790</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>821</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>temp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>temp_r</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fuel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fuel_r</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>421</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
